From 72a8dbffdc43dc73ceb13255ab9c5e861798dbfd Mon Sep 17 00:00:00 2001
From: Kieth Liu <kieth.liu@amlogic.com>
Date: Thu, 18 Jun 2015 19:07:11 +0800
Subject: [PATCH 5658/5965] PD#108029: defconfig: g9tv,g9bb: compile tuner
 si2151 as module

CONFIG_AM_SI2151=m

tvservice will execute the commond below:

echo autoload > /sys/class/amlfe/setting

that done by showbo.zhang@amlogic.com

Change-Id: I19b468c712ea8016ec2abe90ccc9e90288c80399
---
 arch/arm/configs/mesong9bb_defconfig | 2 +-
 arch/arm/configs/mesong9tv_defconfig | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/mesong9bb_defconfig b/arch/arm/configs/mesong9bb_defconfig
index 23fff2a8824b..7895056316a4 100644
--- a/arch/arm/configs/mesong9bb_defconfig
+++ b/arch/arm/configs/mesong9bb_defconfig
@@ -214,7 +214,7 @@ CONFIG_AM_VECM=y
 CONFIG_AM_DVB=y
 CONFIG_AM_ATVDEMOD=y
 CONFIG_AM_DTVDEMOD=y
-CONFIG_AM_SI2151=y
+CONFIG_AM_SI2151=m
 CONFIG_MALI400=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
diff --git a/arch/arm/configs/mesong9tv_defconfig b/arch/arm/configs/mesong9tv_defconfig
index a8b2fb40d695..92399a24c637 100644
--- a/arch/arm/configs/mesong9tv_defconfig
+++ b/arch/arm/configs/mesong9tv_defconfig
@@ -216,7 +216,7 @@ CONFIG_AM_VECM=y
 CONFIG_AM_DVB=y
 CONFIG_AM_ATVDEMOD=y
 CONFIG_AM_DTVDEMOD=y
-CONFIG_AM_SI2151=y
+CONFIG_AM_SI2151=m
 CONFIG_MALI400=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
-- 
2.19.0

